appsdk/sdk.go

```diff
@@ -91,25 +85,22 @@ type AppFunctionsSDK struct {
 	// except when &[]byte{} is specified. In this case the []byte data is pass to the first function in the Pipeline.
 	TargetType                interface{}
 	transforms                []appcontext.AppFunction
-	configProfile             string
-	configDir                 string
-	useRegistry               bool
 	skipVersionCheck          bool
-	overwriteConfig           bool
 	usingConfigurablePipeline bool
 	httpErrors                chan error
 	runtime                   *runtime.GolangRuntime
 	webserver                 *webserver.WebServer
 	edgexClients              common.EdgeXClients
 	registryClient            registry.Client
-	config                    common.ConfigurationStruct
+	config                    *common.ConfigurationStruct
 	storeClient               interfaces.StoreClient
 	secretProvider            *security.SecretProvider
 	storeForwardWg            *sync.WaitGroup
 	storeForwardCancelCtx     context.CancelFunc
 	appWg                     *sync.WaitGroup
 	appCtx                    context.Context
 	appCancelCtx              context.CancelFunc
+	deferreds                 []bootstrap.Deferred
```

Comment:
can we do better with the name here? like deferredFunctions? deferredItems? deferredSomethings?

Reply:
:-) Yea I struggled with that. Had deferedFuncs at one point :-) `deferredFunctions` it will be.